packages/forma-36-react-components/src/components/Asset/Asset.css

```diff
@@ -62,6 +63,8 @@
     1rem * (35 / var(--font-base-default))
   ); /* 35 value comes from font-size multipled by line height multipled by 2 (for limiting text to 2 lines) */
   word-wrap: break-word;
+  padding-left: var(--spacing-s);
+  padding-right: var(--spacing-s);
```

Comment:
I think the padding belongs to `.Asset__asset-container__title` line [#78](https://github.com/contentful/forma-36/pull/569/files#diff-751197682efe2018075d26f9d0715ba4R78)

`.Asset__title-container__title` targets another variant that already has padding

<img width="295" alt="Screenshot 2020-09-21 at 22 48 12" src="https://user-images.githubusercontent.com/6163988/93820863-13f26700-fc5e-11ea-8cad-ed1efb80caf7.png">

The naming isn't so great for this one 🙃

Reply:
Updated